tests/eas/heavy_load.py

```diff
@@ -0,0 +1,96 @@
+#    Copyright 2017-2017 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bart.common.Utils import select_window
+
+from test import LisaTest, experiment_test
+
+WORKLOAD_DURATION_S = 5
+
+REQUIRED_CPU_ACTIVE_TIME_PCT = 99
+
+class HeavyLoadTest(LisaTest):
+    """
+    Test an EAS system under heavy load
+
+    Runs N 100% RT-App threads where N is the number of CPUs, and checks that
+    those tasks were spread across all CPUs in the system by asserting that all
+    CPUs were fully utilized up until the first task completed.
+    """
+
+    test_conf = {
+        'ftrace' : {
+            'events' : ['cpu_idle', 'sched_switch', 'cpu_frequency'],
+        },
+        'modules' : ['cgroups'],
```

Comment:
By removing it, I've noticed that it's required by the executor to freeze the userspace.
Are we know enforcing that behaviour for all tests?... I think we should have a fallback mode for all the cases where, for example, a platform does not support CGroups and/or the freezer is not available...

Otherwise, if it's really required, we should find a way to enable this module by default at executor level (i.e. adding it to the list of modules to be loaded) thus avoiding some "mandatory" lines in the tests sources.